CHANGELOG.md

```diff
@@ -87,6 +87,7 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 - Fix version comparison in install.sh [#3901](https://github.com/chaos-mesh/chaos-mesh/pull/3901)
 - Fix stuck dashboard updates when using ReadWriteOnce PVCs [#3876](https://github.com/chaos-mesh/chaos-mesh/issues/3876)
 - Fix MySQL NO_ZERO_IN_DATE by using `*time.Time` to represent finish time [#4056](https://github.com/chaos-mesh/chaos-mesh/pull/4056)
+- Fix remote cluster cannot upgrade helm release [#4075](https://github.com/chaos-mesh/chaos-mesh/pull/4075)
```

Comment:
Good job!

There is one remaining matter that requires attention. Kindly place this line in the unreleased section above. Additionally, there is no need to force-push the changes repeatedly as this results in the loss of context.